mapswipe_workers/mapswipe_workers/project_types/arbitrary_geometry/grouping_functions.py

```diff
@@ -1,8 +1,6 @@
 import argparse
```

Comment:
I think we can remove all argparse related things here as well. This is just old overhead we don't need anymore. I guess it somehow comes from the times when this script was used directly from the command line. We don't do this anymore and if we want to test the function, we will write a test. So it should be safe to remove it here.